probe.go

```diff
@@ -670,6 +670,121 @@ func probeLicenseStatus(c FortiHTTP) ([]prometheus.Metric, bool) {
 	return m, true
 }
 
+func probeLinkMonitor(c FortiHTTP) ([]prometheus.Metric, bool) {
+	var (
+		linkStatus = prometheus.NewDesc(
+			"fortigate_link_status",
+			"signals if the status is up (1) or down (0)",
+			[]string{"vdom", "link_group", "link"}, nil,
+		)
+		linkLatency = prometheus.NewDesc(
+			"fortigate_link_latency_seconds",
+			"latency of this link in seconds",
+			[]string{"vdom", "link_group", "link"}, nil,
+		)
+		linkJitter = prometheus.NewDesc(
+			"fortigate_link_jitter_seconds",
+			"Electronic jitter on this link in seconds",
+			[]string{"vdom", "link_group", "link"}, nil,
+		)
+		linkPacketLoss = prometheus.NewDesc(
+			"fortigate_link_packet_loss_percentage",
```

Comment:
Prometheus [suggests ](https://prometheus.io/docs/practices/naming/) `_ratio` (0.0-1.0) as the base unit for percentage. I think we should change this to be `x / 100.0` and use `_ratio` to be more conformant.

Reply:
locally already fixed ;)